docs/SECURITY.md

```diff
@@ -0,0 +1,185 @@
+# This document outlines protocol-level security mechanisms
+
+## Custody `IChannel` implementation
+
+### Challenge
+
+HORIZONTAL: what is the last known on-chain state of a channel being challenged
+VERTICAL: what is the status of a channel being challenged
+CHANOPEN: a magic number showing a state is the first deposit state
+CHANCLOSE: a magic number showing a state is final
+
```

Comment:
FYI, it doesn't look like a list in md preview

Reply:
274ff8ee66e2c2032a8095d37d77f7dc87004c16